examples/lighting-app/lighting-common/BUILD.gn

```diff
@@ -38,12 +38,14 @@ if (enable_pw_rpc) {
 }
 
 chip_data_model("lighting-common") {
-  cluster_sources = [
-    "bindings",
-    "level-control",
-    "network-commissioning",
-    "on-off-server",
-  ]
+  zap_file = "lighting-app.zap"
 
   zap_pregenerated_dir = "gen"
+
+  # These clusters are configured in the ZAP file but don't compile.
+  remove_cluster_sources = [
+    "groups-server",
+    "identify",
+    "scenes",
```

Comment:
For lighting app I'd think those should compile.  We should fix them to compile.